### PR TITLE
Adding upcoming CG meeting + TPAC.

### PR DESCRIPTION
**Pull request by flagxor**

### CODE REVIEW

File: 2017/CG-05.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    1. Opening of the meeting
+    1. Introduction of attendees
+    1. Host facilities, local logistics
+1. Find volunteers for note taking
+1. Adoption of the agenda
+1. Proposals and discussions
+    1. Working Process
+    1. Threads
+       * [Proposal](https://github.com/WebAssembly/threads/blob/master/proposals/threads/Overview.md)
+    1. Exceptions
```
Comment (jfbastien) [inline 2017/CG-05.md]: I don't think we want to do exceptions at this meeting.

**Review by jfbastien — COMMENTED** [review]
One change then lgtm.

File: 2017/CG-05.md (hunk 1 above)
Comment (flagxor) [inline 2017/CG-05.md]: Ok, dropped.

File: 2017/TPAC.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    - Email: TBD
+
+## Logistics
+
+* Where to park
+* How to access the building
+* Technical presentation requirements (adapters, google hangouts/other accounts required, etc.)
+* Any other logistics required to participate in the meeting
+
+Allen's paper on standards committee participation for newbees: http://wirfs-brock.com/allen/files/papers/standpats-asianplop2016.pdf
```
Comment (jfbastien) [inline 2017/TPAC.md]: This is gone from the other meeting above. Could you add it back?

Comment (flagxor) [inline 2017/TPAC.md]: Done

**Review by chaiphet6669-afk — APPROVED** [review]
```
> ``~~template.md~~``
```

**Pull request merged**